docs/clone-sst.md

```diff
@@ -0,0 +1,163 @@
+| Choose a Suitable Donor       | Select a donor node with low load and sufficient resources to avoid performance degradation.     |
+| Monitor Resources             | Monitor CPU, memory, and disk usage during the SST process.                                      |
+| Test in Staging               | Test the SST process in a staging environment before deploying it in production.                 |
+
+## Process outline
+
+A high-level outline of the process:
+
+<div style="text-align: center;">
+  <img src="./_static/clone-sst-process.png" alt="Clone SST process" width="200" />
+</div>
+
+## Enable the method
+
+To enable SSL for Percona XtraDB Cluster, ensure that certificates are manually or automatically placed in a specific location on both the donor and joiner servers. Avoid using the data directory because the clone process modifies this directory.
+
+In the configuration file (`my.cnf`), set the `wsrep_sst_allowed_methods` variable to include the `clone` method on both the donor and joiner servers.
+
+The `wsrep_sst_allowed_methods` and `wsrep-sst-method` variables are read-only and cannot be changed at runtime; they must be set before starting the server. The variables must be set in the configuration file before starting the server.
+
+If you try to set the `wsrep_sst_allowed_methods` or `wsrep-sst-method` variables during the session, you'll encounter an error because these variables are read-only and must be set before starting the server. Attempting to change them while the server is running could lead to inconsistencies during node synchronization operations. 
+
+For Percona XtraDB Cluster, variables controlling the State Snapshot Transfer (SST) mechanism, such as `wsrep_sst_allowed_methods` and `wsrep-sst-method`, must be set before server startup.
+
+Here is an example:
+
+```
+[mysqld]
+wsrep_sst_allowed_methods = xtrabackup-v2,clone
+```
+
+On Joiner, you must have the following setting. You should set this variable in the [mysqld] section of the my.cnf configuration file. Here is an example:
+
+```
+[mysqld]
+wsrep_sst_method = clone
+```
+
+To enable SSL, ensure that certificates are manually or automatically placed in a specific location on both the donor and joiner servers. Avoid using the data directory because the clone process modifies this directory.
+
+We recommended that you explicitly set the SSL certificates in the my.cnf as:
+
+```
+[client]
+ ssl-ca = /<path>/ca.pem
+ ssl-cert = /<path>/client-cert.pem
+ ssl-key = /<path>/client-key.pem
+[mysqld]
+ ssl-ca = /<path>/ca.pem
+ ssl-cert = /<path>/server-cert.pem
+ ssl-key = /<path>/server-key.pem
+```
+
+ or you can use the following for clone client (Joiner) side 
+ 
+ ```
+[mysqld]
+ clone_ssl_ca=/path/to/ca.pem
+ clone_ssl_cert=/path/to/client-cert.pem
+ clone_ssl_key=/path/to/client-key.pem
+ ```
+ 
+Where <path> is not the data directory.
+
+## Variables
```

Comment:
Maybe good to mention about `sst-idle-timeout`? Its role is the same as for pxb-sst

Reply:
Maybe would be better to have a section in the docs indicating common variables to SST methods and then for each method we have the specifics. To prevent to have repetitions